sphinx/source/docs/dev_guide/server.rst

```diff
@@ -313,8 +314,8 @@ Some Current Protocol Caveats
    send the whole giant dictionary whenever any entry in it
    changes.
 
-3. At the moment, we do not optimize binary data by sending it
-   over binary websocket frames.
+3. At the moment, we only optimize image data by sending it
+   over binary websocket frames.  All other binary data is unoptimized.
```

Comment:
After reading through #5544, I'm going to revert most of my change from earlier in the WebSocket section and update this to explain base64 encoding of NumPy arrays of type `float32`, `float64` and `int32`.  I'll mention the limitation regarding `int64` types in JS.

I aim to push something later today.  Sorry it's taken me so long to get back to this.